### DISCUSSION TITLE
Cannot find module 'chains/ethereum/dist/_types' when using the Wallet Connect 2.4.7

### DISCUSSION THREAD

**Discussion by bm-kiennguyen** [WalletConnect/walletconnect-monorepo#2734] [Developer Support]
I applied the `"@web3modal/ethereum": "^2.4.7", "@web3modal/html": "^2.4.7",` on Angular and got this error?
Could you please help to check?
![Screen Shot 2023-06-23 at 11 37 15](https://github.com/WalletConnect/walletconnect-monorepo/assets/60868942/a15a6e88-e5bb-4729-89d5-7864918a72c5)

**Comment by glitch-txs** [comment]
Try adding `"skipLibCheck": true` to the ts.config file

**Reply by crypblizz8** [reply]
Thanks @glitch-txs 

Did this help to resolve @bm-kiennguyen ?
